include/arch/arm/arch/machine/gic_common.h

```diff
@@ -37,8 +37,8 @@
  * The layout of IRQs into the CNode are all of PPI's for each core first, followed
  * by the global interrupts.  Examples:
  *   core: 0, irq: 12 -> index 12.
- *   core: 2, irq: 16 -> (2 * 32) + 12
- *   core: 1, irq: 33, (4 total cores) -> (4 * 32) + (33-32).
+ *   core: 2, irq: 16 -> (2 * 32) + 16
+ *   core: 1, irq: 33, (4 total cores) -> (3 * 32) + 33.
```

Comment:
The previous example is clear to me. 4 * 32 clearly expresses that the first 4*32 slots are for the PPI and SGI of the 4 cores.